documentation/Sphinx/user_guide/output/post-processing.rst

```diff
@@ -0,0 +1,100 @@
+.. -----------------------------------------------------------------------------
+     (c) Crown copyright 2024 Met Office. All rights reserved.
+     The file LICENCE, distributed with this code, contains details of the terms
+     under which the code may be used.
+   -----------------------------------------------------------------------------
+
+Post-processing
+---------------
+
+The file 'post-processing.py' can be used for processing the raw outputs 
+into a more usable format, merging all MPI ranks to give the means
+across all ranks.
+
+Please note that you will need an environment with the pandas library. This is 
+included within the Vernier conda environment, and future versions of this
+script will require additional libraries.
+
+.. dropdown:: ``Creating/ loading conda environments in Bash``
+	
+  1. Navigate to the .yml file
+
+  .. code-block:: shell
+
+    cd etc/conda/
+
+  2. Create the environment from the .yml file
+
+  .. code-block:: shell
+
+    conda env create -f vernier-env.yml
+
+  3. Activate the environment
+
+  .. code-block:: shell
+
+    conda activate vernier-env
+
+With an appropriate environment loaded, run the script with:
+
+.. code-block:: shell
+
+    ./post-processing/post-process.py -path=path/to/your/vernier/outputs...
+
+There are a number of command line options that can be passed through the
+script, these are given in the table below.
+
+..  list-table::
+    :widths: 20 15 30
+    :header-rows: 1
+
+    * - Argument
+      - Default
+      - Description
+    * - ``-help``
+      - N.A
+      - Lists all command-line arguments and gives usage
+    * - ``-path``
+      - Current working directory
+      - The path containing the vernier outputs to be processed
+    * - ``-outputname``
+      - vernier-merged-output
+      - Name of file to write to
+    * - ``-inputname``
+      - vernier-output-
+      - Vernier files to read from
```

Comment:
Arguments for executables often follow a standard form which I think we should stick to, in a lot of cases there are two options for the flags, a single letter version preceded by a single `-` and a wordier version preceded by double `--`. The help option already has this (though `-help` also works just because the first letter is h and that is what it sees). For an example of how to do so in argparse see [here](https://github.com/MetOffice/lfric_tools/blob/main/bin/cylcReview#L398).